src/js/models/account/queries/queryBuilders/ProgramActivityQuery.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * ProgramActivityQuery.jsx
+ * Created by michaelbray on 4/17/17.
+ */
+
+const programActivityField = 'program_activity__id';
+
+const spendingOverTimeField =
+    `treasury_account_identifier__program_balances__${programActivityField}`;
+const spendingByCategoryField = programActivityField;
+const awardField = `financial_set__${programActivityField}`;
+
+export const commonQuery = (field, values) => ({
+    field,
+    operation: 'in',
+    value: values
+});
+
+export const buildSpendingOverTimeProgramActivityQuery = (values) =>
```

Comment:
Let's just call these by their endpoints (balances/categories) since to line up with their search operation models name. Otherwise we're dealing with way too many different naming schemes to understand what's happening.